src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -35,75 +35,75 @@
 import org.apache.cassandra.service.CassandraDaemon;
 import org.apache.cassandra.utils.OutputHandler;
 
+import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ThreadPoolExecutor;
+
 import static org.apache.cassandra.tools.BulkLoader.CmdLineOptions;
 
-public class StandaloneScrubber
-{
```

Comment:
Yeah - I noticed *after* I wrote it. (Actually, once I submitted this PR). I'll fix this and the other comments while it's running, since they're not functional issues.